<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="db_label_main">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status[+]</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>254</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_main_menu">
       <property name="text">
        <string>Päävalikko</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset_database">
       <property name="text">
        <string>Alusta tietokanta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>Lopeta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>281</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Kirjautuminen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Käyttäjänimi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_username"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Salasana</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Status[+]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>10</y>
        <width>78</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Opiskelija</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>262</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Opiskelija</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rekisteröidy kurssille</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="id_course_std_0"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_register_std">
         <property name="text">
          <string>Rekisteröidy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Lisää kurssisuorite</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Kurssin tunnus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="id_course_std_1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Suorite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="task_name_std"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Comment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="comment_std"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add_task_std">
         <property name="text">
          <string>Lisää suorite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Arvionnit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Tulosta yksittäiset tehtäväsi tai kurssisuorituksesi</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Kurssin tunnus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="id_course_std_2"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="search_btn_std">
         <property name="text">
          <string>Etsi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_grade_btn">
         <property name="text">
          <string>Tulosta opintosuoritukset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>401</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>351</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Liity tai poistu kurssin opettaja-listalta</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>Instanssin tunnus</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="id_course_tch_0"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QComboBox" name="choice_box">
                 <item>
                  <property name="text">
                   <string>Liity</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Poistu</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="add_btn">
                 <property name="text">
                  <string>OK</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Etsi suoritteita</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Opiskelija (id)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_std_tch_0"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Instanssin tunnus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_course_tch_1"/>
             </item>
             <item>
              <widget class="QPushButton" name="search_assgn">
               <property name="text">
                <string>Etsi suoritteita</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Arvostele kurssisuoritus - täytä kaikki kentät!</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Kurssi</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_course_tch_2"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Suorite</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_exercise_std_0"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Opiskelija</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_std_tch_1"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Kommentti</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="comment_tch"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Arvosana</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="grade_0"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_grade_0">
             <property name="text">
              <string>Arvostele kurssisuorite</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Arvostele kurssi</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Kurssin tunnus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_course_tch_3"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Opiskelija</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_std_tch_2"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Arvosana</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="grade_1"/>
             </item>
             <item>
              <widget class="QPushButton" name="btn_grade_1">
               <property name="text">
                <string>Arvostele</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="changetoStd">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Oppilas-näkymään</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
